package.json

```diff
@@ -13,7 +13,7 @@
         "reconstruct": "npx rimraf node_modules packages/*/node_modules && npm run setup && echo done",
         "setup": "pnpm install",
         "start": "pnpm build && pnpm --recursive --parallel start",
-        "test": "pnpm --recursive test",
+        "test": "turbo run test",
```

Comment:
Now I want this in the admin 👀

Reply:
turbo is already in the admin, it would be very easy to add it for the test command!

Reply:
I think I will 😎